Create epic-css.json

jeremyfaucher opened the pull request:
Thank you!

jeremyfaucher commented:
Is there anything I missed or I need to do to complete pr?

Snapstromegon commented:
Hey @jeremyfaucher 
I think this can be merged.

I want to make some notes though:
- README.md files help users understand how your project works. This is especially important for starters, since most people will have no experience with 11ty (https://github.com/11ty/eleventy-base-blog as an example)
- The epic-css package
  - if you distribute a bundle, you should not distribute the original source
  - if you link to a repo, it should actually hold the source
- Your starter includes google fonts. IMO you should mention that in the README, since that limits usage of your starter for GDPR compliant pages (you can't simply use gfonts in the EU without the user accepting it first).

jeremyfaucher commented:
Hey @Snapstromegon
Thank you for the merge and notes.

-  I updated the README.md and followed the 11ty base theme example. [https://github.com/jeremyfaucher/epic-css-11ty-theme](https://github.com/jeremyfaucher/epic-css-11ty-theme)
- The epic-css package
    -  "you should not distribute the original source". Working on compiling bundle for distribution, hoping to have this in the next version update. 
    -  "if you link to a repo, it should actually hold the source".  Updated repo source link. [https://github.com/jeremyfaucher/epic-css](https://github.com/jeremyfaucher/epic-css)
-  Updated the theme to use System UI font instead of google fonts.

Thank you again for help!

jeremyfaucher commented:
Got it thanks, will remove the build folder as it was just for testing.
